### PR TITLE
Document releasing a new version to EPEL

### PR DESCRIPTION
**Pull request by bocekm**

### CODE REVIEW

**Review by TomasTomecek — APPROVED** [review]
nicely done!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,4 +29,56 @@ $ pip install -r requirements.txt
...
+$ VER=<the_version_released_in_GitHub>  # e.g. VER=0.12
+$ # replace the <your_FAS_user_name> below with your FAS user name
+$ kinit <your_FAS_user_name>@FEDORAPROJECT.ORG  # see [1] on setting up Kerberos credentials
+$ fedpkg clone convert2rhel convert2rhel-distgit  # to get the fedpkg utility, see [2]
+$ cd convert2rhel-distgit
+
+$ fedpkg switch-branch el6  # for EPEL 6
+$ fedpkg pull  # make sure you work with the latest branch content
+$ rm -rf convert2rhel*  # remove all the files related to the previous release
+$ wget https://github.com/oamg/convert2rhel/archive/v${VER}/convert2rhel-${VER}.tar.gz
```
[the remark this answers is not in the document]
Comment (bocekm) [reply CONTRIBUTING.md:59]: Interestingly, the command does not download the tarball for me so I suppose I'll stick with wget :)
```
$ spectool -A *.spec
Source0: https://github.com/oamg/convert2rhel/archive/v0.12/convert2rhel-0.12.tar.gz
$ echo $?
0
$ ll
total 32
drwxr-xr-x.  3 mbocek mbocek 4096 Sep  3 10:50 .
drwxr-xr-x. 43 mbocek mbocek 4096 Aug 26 15:08 ..
-rw-rw-r--.  1 mbocek mbocek 7195 Sep  3 10:48 convert2rhel.spec
drwxr-xr-x.  8 mbocek mbocek 4096 Sep  3 10:48 .git
-rw-r--r--.  1 mbocek mbocek   51 Aug 19 17:48 .gitignore
-rw-r--r--.  1 mbocek mbocek   40 Jan 13  2020 README.md
-rw-r--r--.  1 mbocek mbocek  165 Aug 19 17:48 sources
```

[the remark this answers is not in the document]
Comment (bocekm) [reply CONTRIBUTING.md:59]: Ah, ok, it was missing the -g switch.
```
$ spectool -g -A *.spec
Getting https://github.com/oamg/convert2rhel/archive/v0.12/convert2rhel-0.12.tar.gz to ./convert2rhel-0.12.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   124  100   124    0     0    770      0 --:--:-- --:--:-- --:--:--   765
100 66149    0 66149    0     0   135k      0 --:--:-- --:--:-- --:--:--  135k
```

**Comment by fellipeh** [comment]
LGTM

**Review by fellipeh — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply CONTRIBUTING.md]: interesting :D I never used the -g option, one learns something new every day, thanks!

**Pull request merged**